src/app/components/event-card.tsx

```diff
@@ -25,7 +26,7 @@ export function EventCard(props: EventCardProps): JSX.Element {
       <Stack p="base-loose" spacing="base-loose">
         <SpaceBetween position="relative">
           <Text fontSize={2} fontWeight={500}>
-            {title}
+            <span data-testid={SendFormSelectors.TransferMessage}>{title}</span>
```

Comment:
You don't need to add the `span` here, you can just add the `data-testid` to the `<Text />` component itself.

Reply:
@timstackblock curious why these comments are being resolved with no response or changes? Can you explain why we need to add the `<span />` here?

Reply:
Its needed for my test to pass

Reply:
Why would extra markup be necessary? a data attribute can be added to the parent tag, right?

Reply:
It isn't necessary. If you look in the UI library the `Text` component is a span:

https://github.com/hirosystems/ui/blob/c9efa75863b29b5f30302b38009dded893cf9b0c/packages/ui/src/text.tsx#L6